lib/merch_calendar/date_calculator.rb

```diff
@@ -98,6 +98,18 @@ def julian_to_merch(julian_month)
       end
     end
 
-
+    def merch_months_in(start_date, end_date)
+      merch_months = []
+      prev_date = start_date - 2
```

Comment:
Have you thought about replacing the "magic" numbers (2 and 14) with a more descriptive variable?